### PR TITLE
Fix auth to redirect back to front-end

### PR DESCRIPTION
**Pull request by ARtoriouSs**

### CODE REVIEW

File: lib/squadster_web/controllers/session_controller.ex [hunk 1]
```diff
@@ -20,16 +20,16 @@ defmodule SquadsterWeb.SessionController do
   def callback(%{assigns: %{ueberauth_auth: auth}} = conn, _params) do
     case User.find_or_create(auth) do
       {:ok, user} ->
-        conn
-        |> put_status(:created)
-        |> json(%{message: "Logged in", token: user.auth_token})
+        redirect(conn, external: "#{auth_callback_url}?message=logged_in&token=#{user.auth_token}")
```
Comment (mirterious) [inline lib/squadster_web/controllers/session_controller.ex]: Mega Javist told that `#{auth_callback_url}?message=logged_in&token=#{user.auth_token}` can be moved to const

File: lib/squadster_web/controllers/session_controller.ex [hunk 2]
```diff
@@ -20,16 +20,16 @@ defmodule SquadsterWeb.SessionController do
...
       {:error, reason} -> send_auth_error(conn, reason)
     end
   end
 
   defp send_auth_error(conn, reason) do
-    conn
-    |> put_status(:unauthorized)
-    |> json(%{error: reason, message: "Failed to authenticate"})
+    redirect(conn, external: "#{auth_callback_url}?message=error&reason=#{reason}")
```
Comment (mirterious) [inline lib/squadster_web/controllers/session_controller.ex]: And maybe this thing too

**Review by mirterious — CHANGES_REQUESTED** [review]

File: lib/squadster_web/controllers/session_controller.ex (hunk 1 above)
Comment (ARtoriouSs) [reply lib/squadster_web/controllers/session_controller.ex]: Spravedlivo, but Elixir doesn't have constants as in java))0
I left it as is in order to replace with some lib for params parsing later. To be something like:
`redirect(conn, external: Url(auth_callback_url, params: %{...}`)

But that makes sense, will move to module variable

**Comment by ARtoriouSs** [comment]
Not constant, but refactored. Unable to use module var for all url here because I cannot pass args to it

**Review by mirterious — APPROVED** [review]

**Pull request merged**